### PR TITLE
USHIFT-354: MicroShift: skip sig-instrumentation tests

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

File: test/extended/prometheus/prometheus.go [hunk 1]
```diff
@@ -71,6 +71,14 @@ var _ = g.Describe("[sig-instrumentation][Late] OpenShift alerting rules", func(
 	var alertingRules map[string][]promv1.AlertingRule
 	oc := exutil.NewCLIWithoutNamespace("prometheus")
 
+	g.BeforeEach(func() {
```
Comment (deads2k) [inline test/extended/prometheus/prometheus.go:74]: please enumerate which tests are being skipped and why an apigroup detection or capability detection (do I have a prometheus route) isn't more appropriate.

**Comment by pacevedom** [comment]
/hold
Need to rework.

**Comment by pacevedom** [comment]
/unhold
Ready for review
/assign @ingvagabund

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**